Makefile

```diff
@@ -47,4 +47,9 @@ clean-docker-images:
 
 clean-docker-all: clean-docker-containers clean-docker-buildcache clean-docker-images
 
-clean-all: clean-docker-buildcache clean-docker-containers
\ No newline at end of file
+clean-all: clean-docker-buildcache clean-docker-containers
+
+test:
+	make start-lumigator-build
+	cd lumigator/python/mzai/backend; SQLALCHEMY_DATABASE_URL=sqlite:///local.db uv run pytest
```

Comment:
maybe pushd/popd? so we end up in the same dir afterwards?

Reply:
Going to merge this for now and can circle back on popd